### PR TITLE
Add ipgeolocation

### PR DESCRIPTION
**Pull request by CipherGirl**
- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

This is a new PR of inactive PR #1500, Referencing Issue #2002

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -655,10 +655,12 @@ API | Description | Auth | HTTPS | CORS |
 | [IP Location](https://ip-api.com/docs) | Find location with ip address | No | No | Unknown |
 | [IP Sidekick](https://ipsidekick.com) | Geolocation API that returns extra information about an IP address | `apiKey` | Yes | Unknown |
 | [IP Vigilante](https://www.ipvigilante.com/) | Free IP Geolocation API | No | Yes | Unknown |
+| [ip-api](http://ip-api.com/) | Find location with ip address | No | No | Unknown |
```
Comment (matheusfelipeog) [inline README.md:658]: Remove this, please, already listed here:

```
| [IP Location](https://ip-api.com/docs) | Find location with ip address | No | No | Unknown |
```

If you want to rename the one listed to ip-api, I can accept that ;)

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your new contribution @CipherGirl ;)

Can you resolve this review, please?

File: README.md (hunk 1 above)
Comment (CipherGirl) [reply README.md]: Oh, I didn't notice that, I removed the one I added :)

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok ✔

Thanks for the update @CipherGirl <3

**Pull request merged**